icicle/backend/cpu/src/field/cpu_vec_ops.cpp

```diff
@@ -329,7 +332,7 @@ class VectorOpTask : public TaskBase
 
   // An array of available function pointers arranged according to the VecOperation enum
   using FunctionPtr = void (VectorOpTask::*)();
-  static constexpr std::array<FunctionPtr, static_cast<int>(NOF_OPERATIONS)> functionPtrs = {
+  static constexpr std::array<FunctionPtr, static_cast<int>(NOF_VECOPS_OPERATIONS)> functionPtrs = {
```

Comment:
VECOPS_OPERATIONS = VEC OPERATIONS OPERATION
Why not NOF_VEC_OPERATIONS?

Reply:
Fixed